de.dlr.sc.virsat.uiengine.ui/src/de/dlr/sc/virsat/uiengine/ui/cellEditor/emfattributes/EListStringCellEditingSupport.java

```diff
@@ -0,0 +1,76 @@
+/*******************************************************************************
+ * Copyright (c) 2008-2019 German Aerospace Center (DLR), Simulation and Software Technology, Germany.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package de.dlr.sc.virsat.uiengine.ui.cellEditor.emfattributes;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import org.eclipse.emf.common.command.Command;
+import org.eclipse.emf.ecore.EAttribute;
+import org.eclipse.emf.edit.command.SetCommand;
+import org.eclipse.emf.edit.domain.EditingDomain;
+import org.eclipse.jface.viewers.ColumnViewer;
+
+import de.dlr.sc.virsat.model.dvlm.roles.Discipline;
+
+/**
+ * Custom cell editing support for handling EList of strings in a table viewer.
+ * Extends EStringCellEditingSupport.
+ * @author ngat_di
```

Comment:
Please remove all author-tags you find in the code... Also if you find tags of others... We decieded that we do not want to have some in the source code as they get outdated guide fast and we can the authors by using git...

Reply:
The @author tag is still in there?